numismatic/cli.py

```diff
@@ -166,7 +168,6 @@ def listen(state, exchange, assets, currencies, raw_output, batch_size,
            channel, api_key_id, api_key_secret):
     'Listen to live events from an exchange'
     # FIXME: Use a factory function here
-    from .exchanges import BitfinexExchange, LunoExchange
     assets = ','.join(assets).split(',')
```

Comment:
Lastly, importing into a namespace would mean that we don't have to name this BitfinexExchange - instead, we would call it exchanges.Bitfiniex...

BTW: I agree with your comment that we need a factory...

Reply:
I added a factory method for this.

Reply:
It's not as useful yet as the API/signatures aren't consistent across the exchanges yet but it's a start.